build.gradle

```diff
@@ -33,6 +35,23 @@ buildScan {
     licenseAgree = 'yes'
 }
 
+	task cleanShared() {
+		group 'Configuration'
+		description 'Removes ALL shared libraries from the servlet container.'
+		def libs = serverbase+"/shared/lib"
+		FileTree tree = fileTree(dir: libs)
+		delete fileTree(dir:libs , include: '*')
+	}
+
+	 task cleanTomcat() {
+		group 'Configuration'
+		description 'Removes the deployed uPortal from the servlet container.'
+		def uPortal = serverbase+"/webapps/uPortal"
+		printf uPortal
```

Comment:
I changed the statements a bit "Configured deployed uPortal from servlet container at location:  {uPortalDeployDir}"

The reason I did that when you execute cleanShared you will see that statement even though you have not executed cleanTomcat task. Gradle is prints the printf statment during configuration. I do not want to mislead the developer if they see "Removing...." while gradle is only configuring the task